meta/autoid/autoid.go

```diff
@@ -56,6 +60,21 @@ func GetStep() int64 {
 	return step
 }
 
+// SetStep is only used by tests
+func SetStep(s int64) {
+	step = s
+}
+
+// Base implements autoid.Allocator Base interface.
+func (alloc *allocator) Base() int64 {
+	return alloc.base
+}
+
+// End implements autoid.Allocator End interface.
+func (alloc *allocator) End() int64 {
```

Comment:
Why do we need `End` if it the same as `Base`?

Reply:
My fault, it should return end